cmd/ssvnode/encrypted_private_key.json

```diff
@@ -0,0 +1 @@
4524edff141d4c962b7502512c09b5ddcb611afcbb97d20d87dc329bdb85811d1967d2c5add3b3de8eabdd1c0dc2d304ace04d3e8b01edfd345166ffcb941c97287f60eafc4ba4f28fd133603222eee85eb32bebb5515aec15fd49f4742069fc6265aef4cadb04f5864b4e57bfc7cdc2c81ec7cc2ba32b474a6e7d93146c9f39d949056b435409d0616bbd0ade91dd2b1a0ef0705177666c1d9adbb731d217c1a14d2fcf08a2af1bebc75c29ae07b79a59fe40c3093df17f1289973062d279c673e6cb97779eade505a2d7bdce2e231c47a532f4c75ba7f53031842bdfee6f2f1c146d46aeba01f9d67c9a9809d43117e4efc6300c506ccdc495386a2b3c845ccfcc202a3bfa147bf4253a92c72cc9178a07152767072286434c032f3913dc1521c9c9f835519e29ab23a860a71afb90257f89b17ed62d0441e8fd2c88aa23233e688d2e1601f8fa4971d0be02d8c18822162b193c141e74846e601bae1666e82ca4444ad92e23bb9f13e52d78b09ba7d86a0a3ec871d3c9c6881b126708618357231c6ebf94de6b265619892f82c409b761041ef5a50d64fc584732662fce5237369aa35e729ba29328d58efde0d7014931ffd12affa0ab56142c8e2936817065a0c5166197a3a25cde51b9590856bb9ed4b643729293212cabad3164a0d94dbeee8237807bcb071fadc890e03ced489783ff5d8ee822870930f77515700a8bb43ecfc700f7f7871847feb67602f98508251cdc5a4f81bdd749d8adf9251481854c317ea6c7afd468cbb3f8c97a0555292a8a2103bb5f0c29c741a51a18386ced7b1473b1a88add5fb510267d5fa1a98e76be28d71df7e41ea949659d5e9bfab215e97376d36a27f8895b16deed3a60e88f3b66603436fe45f6801584cac563b1c6045e1632ae70b81838f9d81c749a6a56bdd01d64c02f0f487598f4105d5d76801b913835a333f369c00b35c5bc86eccc8b5ce73db2efd9ed4737a473bff480190d608df5e8b3a319dc9b0cfcb1a2e21f1d0f244e11b6334c440c2f1e8496d446dcaf67b6f3b261d312fccb3b99ff70c18831f7d2e0427f6f0782235da33f953bfa0b4cf6fb55a399d43538325a21c302357c1fd610f7a10b80d065287839aaecd5659492422e9c55583849fe7ac1472083c9fe040129fdabe7c3f1e34275a69e9a21741be8826c5d3a2d1c75655e2debf869a8af1b15a56cfab6221ece47c4f1d4ec2fb94214d0a9bd94c47b52c5b31f5d566ac6a6d95adfc809c77ccacfdb6dbd8f23956518dda8a49448dc778f757a5db5fc186be15db1df0839290aded1b708653c7c318aca956430c5536defc06e4890bc79cfe8bcb3a46cba6721a13a36d008b15f4fd946808b4a328853f68f1a2eec23ba069b326953d9fb8e9792f39cbf852f1d4ab315c5ce959adbbd3fa3772654dfa065c8737cd4dbdd400f4e895300b013040509a4ab18161919feb0dc4fc03d42d74d019c9febc049773147f3c632d207d5a4a24dabb57c49434a917ad36612801e70f2461111e640da8a28017262a4b47d55e0eb8ca8d503d507ecc0e600c6feb9c127fdde7b2857b7556ba39361bf8982c15d634daf8d700c6373b90559a44183d90fc540f63b7df38fd210e3e5a4ef850b38b5e64d6779e6e5cb52ef0bbc2832263a49ac5dc14173886d119cd2d473f2d8de49c4ca30fb34fc9cbd3902d68aee4b85022e1d02ec4901049dcb7230064e9d2c8372a8db834dc4badeaf530dad0702d84ade68dfbaab1a90a440fcd30f2b4770ff68f1bd95144fca466f7ae5e8ec5fcf1e380fc9a1f63ab69bb6db7d71659c66859d5fa0924749e8a04fa95f323cdc01693f6b60ecb665e505322415e73d17c48dda4a9743a67a66dd141d6bbc61d0255b415dc2651de35c05d6fdde053a7e41f0c14dbca632787e6f875904ece3dca0e2e2c59d03b19fadc10ac67808a397c6043876b3e2332c7031","params":{"iv":"eb331a7a7cb65279fb4de531a5153c11"}},"kdf":{"function":"scrypt","message":"","params":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"13ebe5549cb03ca09e0c88a98032b1d1876f42a997e9c02775a9393857d2eec7"}}}
```

Comment:
Let's not commit the private key and add the file mask to `.gitignore` to make sure we don't commit it in the future